README.md

```diff
@@ -9,18 +10,18 @@
 | gpu-unit-test | [![Build Status](https://dev.azure.com/msrasrg/SuperBenchmark/_apis/build/status/cuda-unit-test?branchName=main)](https://dev.azure.com/msrasrg/SuperBenchmark/_build/latest?definitionId=80&branchName=main) |
 
 
-SuperBench is a benchmarking and diagnosis tool for AI infrastructure,
-which supports:
-* Comprehensive AI infrastructure validation
+**SuperBench** is a validation and profiling tool for AI infrastructure, which supports:
+
+* AI infrastructure validation and diagnosis
     * Distributed validation tools to validate hundreds or thousands of servers automatically
     * Consider both raw hardware and E2E model performance with ML workload patterns
-    * Provide a fast and accurate way to detect and locate hardware problems
-    * Performance/Quality Gates for hardware and system release
-* Benchmarking with typical AI workload patterns
+    * Build a contract to identify hardware issues
+    * Provide infrastructural-oriented criteria as Performance/Quality Gates for hardware and system release
+    * Provide detailed performance report and advanced analysis tool  
+  
```

Comment:
remove the extra line to format it as one list

Reply:
sure. removed.